.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ workflows:
       - contract_hackatom
       - contract_ibc_reflect
       - contract_ibc_reflect_send
-      - contract_floaty
+      # - contract_floaty # This contract needs nightly Rust to compile
```

Comment:
Can we just set the Rust version to nightly in the `contract_floaty` implementation?

Reply:
Also could you remove the exception

```
      - run:
          name: Check development contracts
          command: |
            echo "Checking all contracts under ./artifacts"
            docker run --volumes-from with_code rust:1.67.0 \
              /bin/bash -e -c 'export GLOBIGNORE="artifacts/floaty.wasm"; cd ./code; cargo run --bin cosmwasm-check artifacts/*.wasm'
            docker cp with_code:/code/artifacts .
 ```
 
 further down in the file?

Reply:
The only problem I am having with this is that I didn't find an official pinned image. There's [this one](https://hub.docker.com/r/rustlang/rust/), but it is updated every day.

Reply:
Hmm, yeah, true. It creates a risk of unrelated CI fails due to compiler issues. Okay let's keep it as is then.

----

Here we will get a different issue which I initially missed:

```

      - run:
          name: Build development contracts
          command: |
            echo "Building all contracts under ./contracts"
            docker run --volumes-from with_code cosmwasm/rust-optimizer:0.12.13 ./contracts/*/
      - run:
          name: Check development contracts
          command: |
            echo "Checking all contracts under ./artifacts"
            docker run --volumes-from with_code rust:1.67.0 \
              /bin/bash -e -c 'export GLOBIGNORE="artifacts/floaty.wasm"; cd ./code; cargo run --bin cosmwasm-check artifacts/*.wasm'
            docker cp with_code:/code/artifacts .
 ```
 
 The first step will fail for floaty as `cosmwasm/rust-optimizer:0.12.13` uses Rust stable. So we need to more the exception from the checking step to the build step.